apps/condo/domains/property/schema/Property.js

```diff
@@ -51,38 +91,6 @@ const Property = new GQLListSchema('Property', {
             isRequired: false,
         },
 
-        address: {
```

Comment:
What about other places? Resident?

Reply:
The plugin may be applied to any model. Of course, I'll add it to the `Resident` model if needed.